pages/common/mongorestore.md

```diff
@@ -0,0 +1,19 @@
+# mongorestore
+
+> Utility to import a collection or database from a binary dump into a MongoDB instance.
+
+- Import a bson data dump from a folder to a mongodb instance:
+
+`mongorestore --host {{database_host}} --port {{port}} --db {{database_name}} {{path/to/folder}}`
```

Comment:
I feel `--host` and `--port` can be dropped here since they are anyways used in the next examples. Lets lay it slowly on the user :smile: